packages/band-arithmetic/band-arithmetic.js

```diff
@@ -0,0 +1,140 @@
+'use strict';
+
+const _ = require('underscore');
+const parse_georaster = require("georaster");
+
+const get = require('../get/get');
+const utils = require('../utils/utils');
+const convert_geometry = require('../convert-geometry/convert-geometry');
+const intersect_polygon = require('../intersect-polygon/intersect-polygon');
+
+const logger = require('../../logger');
+const parse = require('mathjs').parse;
+
+const regex_multi_character = /[A-z]{2}/g;
+
+const is_leaf_node = node => !node.op;
+
+const variables = [...Array(13)].map((val, i) => String.fromCharCode(i + 65).toLowerCase());
+const operations = {
+  add: (a, b) => a + b,
+  subtract: (a, b) => a - b,
+  multiply: (a, b) => a * b,
+  divide: (a, b) => a / b
+};
+
+const get_value = (input, band_values) => {
+  if (input.value) return input.value;
+
+  const variable_index = variables.findIndex(variable => variable === input.name);
+  return band_values[variable_index];
+};
+
+const compute_value = (node, band_values) => {
+  const left_node = node.args[0];
```

Comment:
Super trivial, but can you do array destructuring here?  I'm not sure tho...  I'll still approve regardless